packages/nimble-components/src/table/specs/table-columns-hld.md

```diff
@@ -291,61 +323,10 @@ const numberWithUnitCellView = NumberWithUnitCellView.compose({
 });
 ```
 
-Here we have a cell visualized in different ways based on custom logic:
+#### Exposing cell view events externally
 
-```TS
-type TableColumnPositiveNegativeNumberCellData = NumberField<'value'>;
-
-const isPositive = (value: number): bool => {
-    return value >= 0;
-}
-
-public class TableColumnPositiveNegativeNumber extends TableColumn {
-    ...
-
-    public cellRecordFieldNames = ['value'] as const;
-
-    @attr
-    public valueKey: string;
-
-    public cellViewTag = 'table-cell-view-positive-negative-number';
-
-    public getDataRecordFieldNames(): string[] {
-        return [valueKey];
-    }
-
-    public validateCellData(cellData: TCellRecord): void {
-        if (typeof(cellData['value']) !== 'number') {
-            throw new Error('Type for cellData is incorrect!');
-        }
-    }
-}
-
-class PositiveNegativeNumberCellView extends TableCellView<TableColumnPositiveNegativeNumberCellData> {
-    public get textFieldCssClass(): string {
-        return this.cellRecord.value > 0 ? 'good' : 'bad';
-    }
-}
-const positiveNegativeNumberCellView = PositiveNegativeNumberCellView.compose({
-    baseName: 'table-cell-view-positive-negative-number',
-    template: html<PositiveNegativeNumberCellView>`
-        <nimble-text-field
-            class="${x => x.textFieldCssClass}"
-            readonly="true"
-            value="${x => x.cellRecord.value}"
-        >
-        </nimble-text-field>`,
-    styles: css`
-        .good {
-            color: green;
-        }
-
-        .bad {
-            color: red;
-        }
-    `
-});
-```
+<!--
```

Comment:
I commented out this part of the spec because it talks about a pattern for responding to events from your cell view instance that is actively being discussed in #1153 
Responding to anchor tag clicks is exactly the use-case this part of the spec was trying to consider. As part of #1153 we should update this part of the spec for event handling column view @m-akinc